src/main/java/uk/gov/pay/api/model/PaymentForSearchResult.java

```diff
@@ -12,9 +12,9 @@
     private PaymentLinksForSearch links = new PaymentLinksForSearch();
 
     public PaymentForSearchResult(String chargeId, long amount, PaymentState state, String returnUrl, String description,
-                                  String reference, String paymentProvider, String createdDate,
+                                  String reference, String email, String paymentProvider, String createdDate,
```

Comment:
This constructor's arguments are ever growing now, 14 in total, we need to think of an alternate at some point (builder?), but may be for a later refactoring task. Fine as such.